util/package.go

```diff
@@ -78,7 +78,10 @@ type Version struct {
 }
 
 type Image struct {
-	Src   string `config:"src" json:"src" validate:"required"`
+	// Src is relative inside the package
+	Src string `config:"src" json:"src" validate:"required"`
+	// Path is the absolute path in the url
+	Path  string `config:"path" json:"path"`
```

Comment:
@jfsiii If we go forward, this would be a breaking change on the Kibana side. Also ideas around "correct" naming?

Reply:
Perhaps `pathname` vs `path` as in `url.pathname` https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname & https://nodejs.org/docs/latest-v12.x/api/url.html#url_url_pathname

Reply:
`path` is pretty much in line with the other places we use it like for downloads: https://github.com/elastic/package-registry/blob/master/docs/api/package.json#L10 If we adjust it, we should adjust it everywhere.

Perhaps we can rename `src` to `local_path`, `relative_path` or something like this?